### DISCUSSION TITLE
What do "Lock Alpha" and "Dither" do in the Pixel brush?

### DISCUSSION THREAD

**Discussion by Murnjendoof** [bitbof/klecks#78] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/55724715/236588895-9ea4246b-4057-4212-8f5c-1cc89e1ae82c.png)
"Eraser" is pretty self-explanatory but I have no idea what the other two options do.

**Comment by bitbof** [comment, marked as answer]
When you draw with lock alpha, it will only affect the rgb channels, not the alpha channel. Meaning, when you draw on transparent pixels, they stay transparent. It's useful for changing the color of something without changing its shape.

https://user-images.githubusercontent.com/870828/236611469-7ce6a64f-c8e1-46c6-8897-3343694a556c.mov


Dither becomes noticeable when you change the opacity. It will only draw on a certain amount of pixels (according to the opacity) following a pattern. Through that you can achieve color gradients with less colors. More on dithering in pixel art https://pixelparmesan.com/dithering-for-pixel-artists/

 
https://user-images.githubusercontent.com/870828/236611486-0bcbf651-30b2-4882-bdc1-f6289ac14353.mov

**Comment by Murnjendoof** [comment]
I see, thank you very much!
